src/setup.ts

```diff
@@ -57,8 +59,14 @@ async function setup(): Promise<void> {
         }
     }
 }
-  
-export = setup;
+
+// notationCLIVersion returns the semantic version of notation CLI
+export const notationCLIVersion = async (): Promise<string> => {
+    const {stdout: stdout} = await exec.getExecOutput('notation', ['version']);
+    let versionOutput = stdout.split("\n");
+    return String(semver.clean(versionOutput[2].split(":")[1].trim()));
+}
```

Comment:
Can you have a better way to extract `version` as the `notation version` may have more fields in the future and they can be before the `version` field.

Reply:
Updated.